corpus/convert.py

```diff
@@ -79,11 +81,22 @@ def run(self):
 
 
 class CorpusToTxtJob(Job):
-    def __init__(self, corpus_path):
+    """
+    Extract orth from a Bliss corpus and store as raw txt or gzipped txt
+    """
+    def __init__(self, corpus_path, segment_file=None, gzip=False):
+        """
+
+        :param Path corpus_path: Bliss corpus
+        :param Path segment_file: segment file
+        :param bool gzip: gzip the output text file
+        """
         self.set_vis_name("Extract TXT from Corpus")
 
         self.corpus_path = corpus_path
-        self.txt_path = self.output_path("corpus.txt")
+        self.gzip = gzip
+        self.segment_file = segment_file
+        self.out_txt = self.output_path("corpus.%s" % (".txt.gz" if gzip else ".txt"))
```

Comment:
creates two dots in the filename, better:
`self.out_txt = self.output_path("corpus.txt" + ".gz" if gzip else ""))`